src/Auth0.ManagementApi/Models/HookUpdateRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Auth0.ManagementApi.Models
+{
+    public class HookUpdateRequest : HookBase
```

Comment:
`HookBase` includes `triggerId`, but the API does not support updating `triggerId`. https://auth0.com/docs/api/management/v2#!/Hooks/patch_hooks_by_id

Reply:
Moved `triggerId` into `HookCreateRequest`